projects/ngx-openlayers/src/lib/sources/xyz.component.ts

```diff
@@ -41,15 +43,15 @@ export class SourceXYZComponent extends SourceComponent implements AfterContentI
   @Input()
   maxZoom: number;
   @Input()
-  tileGrid: tilegrid.TileGrid;
+  tileGrid: any;
   @Input()
-  tileLoadFunction?: TileLoadFunctionType;
+  tileLoadFunction?: any;
   @Input()
   tilePixelRatio: number;
   @Input()
   tileSize: number | Size;
   @Input()
-  tileUrlFunction: TileUrlFunctionType;
+  tileUrlFunction: any;
```

Comment:
I think that this shoul be `tileUrlFunction: UrlFunction; with `import { UrlFunction } from 'ol/Tile';`